articles/modules/ROOT/pages/realtime-querying-of-the-query-log-files.adoc

```diff
@@ -0,0 +1,85 @@
+= Realtime querying of the query.log files
+// CHANGE THIS, REMOVE COMMENTS BEFORE MERGE
+// ANY SLUG CHANGES WILL CAUSE POSTS TO BE RE-CREATED, BREAKING EXISTING LINKS
+:slug: realtime-querying-of-the-query-log-files
+:author: Dana Canzano
+:twitter: @your-twitter
+// relevant versions
+:neo4j-versions: 4.3
+// see taxonomy in readme, remove this comment
+:tags: query logs
+// category see https://github.com/neo4j-documentation/knowledge-base/blob/master/kb-categories.txt
+:category: cypher
+
+Commencing wth Neo4j 4.3 and with Enterprise Edition, one can choose to have the `logs\query.log` log in traditional text format, i.e. PLAIN, or you can have each record 
```

Comment:
It may make sense to use the unix forward slash instead of the windows backslash for path separators, that would also make it consistent with other path formats.